Blockzilla/UIConstants.swift

```diff
@@ -174,18 +174,16 @@ struct UIConstants {
     struct layout {
         static let browserToolbarDisabledOpacity: CGFloat = 0.3
         static let browserToolbarHeight: CGFloat = 44
-        static let copyButtonAnimationDuration: TimeInterval = 0.1
         static let deleteAnimationDuration: TimeInterval = 0.25
         static let alphaToZeroDeleteAnimationDuration: TimeInterval = deleteAnimationDuration * (2 / 3)
         static let displayKeyboardDeleteAnimationDuration: TimeInterval = deleteAnimationDuration * (1 / 3)
         static let lockIconInset: Float = 4
         static let navigationDoneOffset: Float = -10
         static let overlayAnimationDuration: TimeInterval = 0.25
         static let overlayButtonHeight: Int = 56
-        static let numberOfSearchSuggestions: Int = 5
+        static let numberOfSearchSuggestions: Int = 4
```

Comment:
This change should be uncommitted since it's being handled in a separate PR now :)